src/store/NotesSlice.js

```diff
@@ -8,10 +8,19 @@
 export default function createNotesSlice(set, get) {
   return {
     notes: null,
+    createNote: false,
+    createdNotes: null,
+    deletedNotes: null,
+    synchNotes: true,
     comments: null,
     selectedNoteId: null,
     selectedNoteIndex: null,
     setNotes: (notes) => set(() => ({notes: notes})),
+    toggleSynchNotes: () => set((state) => ({synchNotes: !state.synchNotes})),
+    createNoteOn: () => set((state) => ({createNote: true})),
```

Comment:
Please use IsCondition naming for booleans, esp so createNotesOn becomes setIsNoteCreateActive or smth.  "createNotesOn" is a little confusing